habitat/utils/pickle5_multiprocessing.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import io
+import sys
+from multiprocessing.connection import Connection
+from multiprocessing.reduction import ForkingPickler as _ForkingPickler
+
+if sys.version_info[:2] < (3, 8):
+    # pickle 5 backport
+    try:
+        import pickle5 as pickle
+    except ImportError:
+        import pickle  # type: ignore[no-redef]
```

Comment:
@erikwijmans Only thing I am unsure is what to log here: Should we give a warning? And should we use the `habitat.core.logger` or `warning.warn`

Reply:
Probably logger.  This will only run once anyways